package.json

```diff
@@ -75,13 +74,12 @@
     "prettier": "^1.19.1",
     "react": "^16.12.0",
     "react-dom": "^16.12.0",
-    "rollup": "^1.27.4",
-    "rollup-plugin-babel": "4.3.3",
-    "rollup-plugin-peer-deps-external": "^2.2.0",
+    "rollup": "^1.31.0",
+    "rollup-plugin-babel": "^4.3.3",
+    "rollup-plugin-sourcemaps": "^0.4.2",
     "webpack": "^4.41.5",
     "webpack-cli": "^3.3.10",
-    "webpack-dev-server": "^3.10.1",
-    "webpack-node-externals": "^1.7.2"
+    "webpack-dev-server": "^3.10.1"
```

Comment:
So we still need webpack for the dev server? I'd worry about configs getting out of sync... but maybe they are simple enough that it's not a concern? Projects like [rollup-plugin-livereload](https://www.npmjs.com/package/rollup-plugin-livereload) exist, but can't vouch for it.

Suggestion: if it works for now, great, but file an issue if you think there's something to be cleaned up. (There might not be.)

Reply:
#97